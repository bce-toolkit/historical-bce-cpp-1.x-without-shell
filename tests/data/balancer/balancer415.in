Co(NO3)2+NH4NO3+NH3+O2=Co(NH3)6(NO3)3+H2O