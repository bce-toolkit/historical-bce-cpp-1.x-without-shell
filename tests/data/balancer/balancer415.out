(4Xa)Co(NO3)2+(6Xa-4Xb)NH4NO3+(16Xa+8Xb)NH3+(-3Xa+8Xb)O2=(4Xa)Co(NH3)6(NO3)3+(4Xb)H2O